configure.py

```diff
@@ -849,16 +860,32 @@ def pkg_config(pkg):
 
 
 def try_check_compiler(cc, lang):
-  try:
-    proc = subprocess.Popen(shlex.split(cc) + ['-E', '-P', '-x', lang, '-'],
+  if sys.platform == 'zos':
+    f = tempfile.NamedTemporaryFile(delete=False, suffix=".cc")
```

Comment:
I think this and the below code is assuming njsc as the build compiler.  Should we try with Wyvern (clang) and see if it's still required?  Or add a check based on the `cc`

Reply:
I tried running `configure.py` without this change with njsc and it works, do you remember the reason of this code? were it suppose to create an error without it?

Reply:
I think the v8 version of njsc had an issue with the generated code.  Perhaps they fixed it in later versions of njsc.

Reply:
I will remove this code as it build fine without it